services/secondary-storage/server/test/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResourceTest.java

```diff
@@ -88,6 +93,19 @@ public void testMount() throws Exception {
         }
     }
 
+    @Test
+    public void testCleanupNfsStaging(){
+        TemplateObjectTO templateMock = Mockito.mock(TemplateObjectTO.class);
+        Exception exception = new Exception();
+        resource.logger = Mockito.mock(Logger.class);
+
+        Mockito.doNothing().when(resource.logger).debug("Failed to clean up staging area:", exception);
+        Mockito.when(resource.execute(any(DeleteCommand.class))).thenThrow(exception);
```

Comment:
I didn't use that other approach because I think it adds more complications. I find it easier to use the Mockito and then just checking if the method was used. It feels more natural.